framework/graphics/fps_info.h

```diff
@@ -58,16 +58,16 @@ class FpsInfo
     [[nodiscard]] uint64_t ShouldPreloadFrames(uint64_t current_frame) const;
 
   private:
-    uint64_t start_time_;
+    uint64_t start_time_{};
 
     uint64_t measurement_start_frame_;
     uint64_t measurement_end_frame_;
 
     int64_t measurement_start_time_;
     int64_t measurement_end_time_;
 
-    uint64_t replay_start_time_;
-    int64_t  replay_start_frame_;
+    uint64_t replay_start_time_{};
+    int64_t  replay_start_frame_{};
```

Comment:
why is this signed? @bradgrantham-lunarg ?

Reply:
yes, why? :)

Reply:
I don't know.  #1907